### PR TITLE
Make card appear in lovelace card picker

### PR DESCRIPTION
**Pull request by dcs8**
This will make the card appear in the Lovelace card picker (I originally made this request against master but realised I should have used Dev branch instead)

### CODE REVIEW

**Comment by agneevX** [comment]
Did you test this successfully?

File: src/main.js [hunk 1]
```diff
@@ -952,3 +952,12 @@ class MiniGraphCard extends LitElement {
...
 
 customElements.define('mini-graph-card', MiniGraphCard);
+
+// Configure the preview in the Lovelace card picker
+window.customCards = window.customCards || [];
+window.customCards.push({
+  type: 'mini-graph-card',
+  name: 'Mini Graph Card',
+  preview: false,
+  description: 'The Mini Graph card is a minimalistic and customizable graph card',
```
Comment (agneevX) [inline src/main.js:962]: Remove the comma.

Comment (dcs8) [reply src/main.js:962]: yes I tested successfully - removing the comma gives a 'missing trailing comma (comma-dangle)' error when running through npm

Comment (agneevX) [reply src/main.js:962]: From the last object, of course. https://eslint.org/docs/rules/comma-dangle

**Review by agneevX — APPROVED** [review]

File: src/main.js (hunk 1 above)
Comment (dcs8) [reply src/main.js:962]: Can see you've approved now, are there any changes required?

Comment (agneevX) [reply src/main.js:962]:
```suggestion
  description: 'The Mini Graph card is a minimalistic and customizable graph card'
```

Comment (dcs8) [reply src/main.js:962]: You might have to bare with me as I'm new to js and npm, so sorry if I'm missing something. But like I said above, removing that comma gives a 'missing trailing comma (comma-dangle)' error when running through 'npm run build'. Comma dangle seems to be set to always-multiline (or possibly always). So my understanding is that no change is required.

**Comment by kalkih** [comment]
Thanks for the PR, looks good 👍

**Pull request merged**